.gitignore

```diff
@@ -50,3 +50,5 @@ docs/_build/
 # IDEA
 .idea/
 *.iml 
+
+.DS_Store
```

Comment:
@georgeyk To follow the format of the rest of the file, maybe put a category comment, such as "OS-specific files" or something.